deepmd/pt/model/task/invar_fitting.py

```diff
@@ -146,40 +138,6 @@ def deserialize(cls, data: dict) -> "GeneralFitting":
         check_version_compatibility(data.pop("@version", 1), 1, 1)
         return super().deserialize(data)
 
-    def compute_output_stats(
-        self,
-        merged: Union[Callable[[], List[dict]], List[dict]],
```

Comment:
@wanghan-iapcm should I remove all fitting net level stat UTs, `tests/pt/test_stat.py`, and `tests/pt/model/test_polar_stat.py`?

Reply:
I agree